beta-src/src/App.tsx

```diff
@@ -1,13 +1,17 @@
 import * as React from "react";
 import "./assets/css/App.css";
 import Box from "@mui/material/Box";
+import { ThemeProvider } from "@mui/material";
+import webDiplomacyTheme from "./webDiplomacyTheme";
```

Comment:
the theme provider wraps app already in index. we should not be placing it here again. pls revert this file.

Reply:
Whoops! I totally missed that lol. File has been reverted!